packages/worker/src/worker.ts

```diff
@@ -432,6 +432,7 @@ export class Worker {
     workflowBundle: WorkflowBundleWithSourceMapAndFilename,
     compiledOptions: CompiledWorkerOptions
   ): Promise<WorkflowCreator> {
+    // TODO: this isn't required, consider running everything in the threaded vm workflow runtime
```

Comment:
I had came up to the same conclusion. We only need `isolateExecutionTimeout` to be set to some high values...

... except if we want to support debugging using Chrome's Dev Tools. Apparently, Chrome Dev Tools doesn't support debugging nodejs threads. But there are some other draw bakcs to Dev Tools. I don't think we should keep that debug target in mind.

Reply:
I guess it doesn't hurt but I'm fine keeping this.
I'm relying on this to access a global variable from workflow context.